GoogleForCreators/web-stories-wp
# Floating Menu: Border Color Eyedropper Regression

## Context

Felipe reported a regression here: #11216

## Summary

Wasn't seeing the same regression as @felipebochehin87  but a change in the borderWidthAndColor was overwritten which was hiding the eyedropper on the floating menu border color picker. 

## Relevant Technical Choices

Just putting the `borderWidthAndColor` color picker back to the props it needs for the eyedropper.  (see https://github.com/GoogleForCreators/web-stories-wp/pull/11075/files)

## To-do



## User-facing changes

Be able to update border color from floating menu color picker with a swatch or the eyedropper or the slider and for it to stay in sync with the border color selected in the style panel. 

## Testing Instructions

(Bug reproduction steps copied from #11216)
1. Open the editor
2. Add an image element
3. Add a border to the element
4. Click on the element and pick a color from Style menu - color is set
5. Click on the element and pick a color from floating menu - color is not set, unless user drags and drops the element 


Selected color in floating menu and in sidepanel should remain in sync. 




This PR can be tested by following these steps:

1.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #11216
